tests/cli/test_build.py

```diff
@@ -46,6 +46,14 @@ def test_build_by_tags():
     )
     cmd.build()
 
+    #  testing multiple tags as comma seperated list:  buildtest build --tags fail,pass --tags python,ping --tags network
+    cmd = BuildTest(
+        configuration=configuration,
+        tags=["fail,pass", "python,ping", "network"],
+        buildtest_system=system,
+    )
+    cmd.build()
```

Comment:
we can cut down on number of tags to avoid lots of builds so it can be sufficient to just do  `tags = ["fail,python", "network"]`.

It would be good exercise to also test for edge cases such as comma separated without any tags or a tagname with a trailing comma

```
buildtest build --tags ,,
buildtest build --tags python,
buildtest build --tags ,python
```